dev/docker-compose.yaml

```diff
@@ -199,6 +201,13 @@ services:
 
   shl-creator:
     image: ghcr.io/uwcirg/shl-ltt:${SHL_CREATOR_IMAGE_TAG:-latest}
+    environment:
+      VITE_API_BASE: https://shl-server.${BASE_DOMAIN}/api
+      VITE_BACK_URL: https://${BASE_DOMAIN}/pro_reports/clinic_report_inform
+      VITE_SOF_ISS: https://fhir-auth.${BASE_DOMAIN}/fhir
+      VITE_OIDC_SERVER_BASE: https://keycloak.${BASE_DOMAIN}
+      VITE_OIDC_LOGOUT_ENDPOINT: https://keycloak.${BASE_DOMAIN}/realms/ltt/protocol/openid-connect/logout
+      VITE_OIDC_CHECK_SESSION_IFRAME: https://keycloak.${BASE_DOMAIN}/realms/ltt/protocol/openid-connect/login-status-iframe.html
```

Comment:
I cant find any references in the keycloak docs to this page, is there someplace I can read more about it?

Reply:
@daniellrgn found an OIDC spec for this to add later

Reply:
OIDC spec regarding use of `login-status-iframe.html` to check session status:
https://openid.net/specs/openid-connect-session-1_0.html